content/06.suppmaterial.md

```diff
@@ -122,6 +124,8 @@ In all cases, the errors affected 1-10 bp.
 
 ### Supplementary Figures
 
+[//]: # (I think the precision-recall curves would be easier to interpret if the limit (size) of each axis were the same. Right now it is a bit hard to interpret the trade-off of each method. In my opinion they to not necessary have to go from 0-1.)
```

Comment:
I agree!!!

Reply:
The tricky thing is that if we zoom in we would lose delly in many graphs, and if we don't it's difficult to see the differences. 

![hgsvc-sim](https://user-images.githubusercontent.com/5704457/57961537-8150f380-78c4-11e9-86a7-e0f752aebb4c.png)

We could show both, zoomed out and zoomed in, both with same scale for precision and recall.

Reply:
For example: 

![hgsvc-sim](https://user-images.githubusercontent.com/5704457/57962789-4654bd00-78d0-11e9-858c-58b70be0ac52.png)

It takes more space and will make the supp longer but it's just the supp. What do you think?

Reply:
You are right it becomes hard to see the difference in precision when the recall goes to almost zero. 

I really like the precision-recall figures with the zoom. It is a good idea showing the whole range (0-1) in the top graphs and then a zoom in the bottom. I personally do not think it is a problem that it takes up more space in the supplementary, since no new figures (and figure references) will need to be added.

Reply:
Thanks for the feedback, I'll change the supp figs to the zoom version. I'll do it in another PR where I cleanup the code for the figures when I added figures for the per-size evaluation.